cluster.go

```diff
@@ -0,0 +1,183 @@
+package cbcolumnar
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/couchbaselabs/gocbconnstr"
+)
+
+type Cluster struct {
+	client clusterClient
+
+	timeoutsConfig TimeoutOptions
+}
+
+func Connect(connStr string, credential Credential, opts ConnectOptions) (*Cluster, error) {
```

Comment:
The Connection RFC is now calling this `NewInstance`, I don't know whether there was a reason behind this change compared to SDK3.